packages/design-system/src/components/ChoiceList/Choice.tsx

```diff
@@ -112,13 +116,16 @@ export const Choice = (props: ChoiceProps) => {
 
   let errorId;
   let errorElement;
-  if (props.errorMessage) {
+  if (!props._choiceChild) {
     errorId = props.errorId ?? `${props.id}__error`;
     errorElement = (
       <InlineError id={errorId} inversed={props.inversed} className={props.errorMessageClassName}>
         {props.errorMessage}
       </InlineError>
     );
+  } else {
+    const { _choiceChild, ...rest } = props;
+    props = rest;
```

Comment:
What's the advantage of this over `delete props._choiceChild`? It's still mutating the pointer `props` even if it doesn't mutate the object. I think delete is a little clearer

Reply:
Using delete caused tests for MonthPicker and ChoiceList to fail. I can't remember what the error was, I think it had to do with a type error where you couldn't delete something from an object.

Reply:
Huh, it's kind of odd that TypeScript would be mad about that but fine with reassigning `rest` to `props` even though it's a slightly different type. If we moved it up to the function signature like

```ts
export const Choice = ({ _choiceChild, ...props }: ChoiceProps) => {
```

then we wouldn't need the `else` block at all.